code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -262,7 +262,7 @@ Borg Shaker
 	recharge_time = 3
 	accepts_reagent_upgrades = FALSE
 
-	reagent_ids = list(/datum/reagent/toxin/fakebeer, /datum/reagent/consumable/ethanol/fernet)
+	reagent_ids = list(/datum/reagent/consumable/ethanol/fernet, /datum/reagent/consumable/ethanol/bastion_bourbon, /datum/reagent/toxin/zombiepowder, /datum/reagent/smoke_powder, /datum/reagent/smoke_powder_activator)
```

Comment:
/datum/reagent/toxin/zombiepowder is insanely good, what the other 2 do?

Reply:
Fernet is an especially strong drink that works similarly to lipolicide - it reduces your nutrition and causes toxin damage if you're starving. This drink was already in the shaker and also available in a limited, but large quantity to bartenders via booze-o-mat. 

Bastion Bourbon provides universal healing at the cost of getting drunk:
> Heals 1 brute, 1 burn, 3 oxy, 0.5 tox, 5 stamina per tick if you aren't in crit. No healing over time in crit. 
> Heals 1 of each basic damage + stamina **per unit** ingested instantly, doubled if crit.

Reply:
Additionally, zombie powder is not very strong since it got nerfed. I want to make sure you aren't remembering it as an instant knockout drug (which is what fakebeer is and why I'm removing fakebeer).